src/Schema/SQLiteSchemaManager.php

```diff
@@ -222,7 +221,9 @@ protected function _getPortableTableForeignKeysList(array $rows): array
 
             // Inferring a shorthand form for the foreign key constraint, where the "to" field is empty.
             // @see https://www.sqlite.org/foreignkeys.html#fk_indexes.
-            $foreignTablePrimaryKeyColumnRows = $this->fetchPrimaryKeyColumns($value['foreignTable']);
+            $foreignTablePrimaryKeyColumnRows = $this->fetchPrimaryKeyColumns(
+                OptionallyQualifiedName::quoted($value['foreignTable']),
+            );
```

Comment:
This is an issue similar to the one fixed in https://github.com/doctrine/dbal/pull/6744 – while parsing introspected results, the schema manager needs to fetch more, which makes it convert the data from the lower-level format (string) to the higher-level format (`OptionallyQualifiedName`) mimicking the user input for the `fetch*()` method.

This isn't nice but isn't the end of the world and could be fixed separately.